src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -123,6 +128,61 @@ export const NoMatchingResults = (): JSX.Element => {
     return <GenericFilterEmptyState />
 }
 
+const ExternalLinkIframeModal = ({ selectedExternalLink, handleCloseModal }) => {
+    const handleOpenExternalLink = () => {
+        window.open(selectedExternalLink.externalLinkURL, '_blank')
+    }
+
+    return (
+        <VisibleModal2 close={handleCloseModal}>
+            <div
+                className="flexbox-col dc__position-abs br-8 dc__top-12 dc__bottom-12 dc__right-12 dc__left-12 bcn-0"
+                onClick={stopPropagation}
+            >
+                <div className="flexbox dc__content-space px-20 py-12 dc__align-items-center dc__border-bottom">
+                    <div className="flexbox dc__gap-8 dc__align-items-center">
+                        <img
+                            className="icon-dim-20"
```

Comment:
Images should always contain height and width attributes so that till the time its not loaded, space for image is reserved saving CLS.
Ref: https://www.ltvco.com/engineering/height-width-attributes-img-tag/